docs/source/client.rst

```diff
@@ -71,6 +63,16 @@ remotely with functions like ``submit``, ``map``, ``get`` and ``compute``.
 See :doc:`efficiency <efficiency>` for more information on efficient use of
 distributed.
 
+We can submit tasks on futures and use futures as inputs. The function will go to 
+the machine where the futures are stored and run on the result once it has completed.
+
+.. code-block:: python
+
+   >>> y = client.submit(inc, x)      # Submit on x, a Future
+   >>> total = client.submit(sum, L)  # Map on L, a list of Futures
```

Comment:
```suggestion
   >>> total = client.submit(sum, L)  # Submit on L, a list of Futures
```

I just noticed, we're `submit`ing sum, not mapping it. This was already present, but can you fix it up too since you're moving this line?

Reply:
okay, will do!